views/index.ejs

```diff
@@ -0,0 +1,84 @@
+<%- contentFor('body') %>
+<h1>mdn-bcd-collector</h1>
+<p>Data collection service for <a href="https://github.com/mdn/browser-compat-data">mdn/browser-compat-data</a>. This website provides automatic feature detection for Web APIs and CSS properties, to assist in obtaining data for the BCD project.</p>
+
+<div>
+  <button id="start">Run</button>
+  <select id="tests">
+    <% tests.forEach(function(test) { %>
+      <option value="<%- test[1] %>"><%- test[0] || 'All Tests' %></option>
+    <% }); %>
+  </select>
+  <br />
+  <div id="individualTestOptionsBox" style="display: none;">
+    <input type="checkbox" id="secureContext"> <label for="secureContext">Use HTTPS?</label>
+    <div id="limitExposureBox" style="display: none;">
+    <label for="limitExposure">Limit Exposure:</label>
+    <select id="limitExposure">
+      <option value="">All Global Exposures</option>
+      <option value="?exposure=Window">Window</option>
+      <option value="?exposure=Worker">Worker</option>
+      <option value="?exposure=ServiceWorker">Service Worker</option>
+    </select>
+  </div>
+</div>
+
+<p>Note: this website is in beta; not all feature detection is accurate yet. If you find any errors, please file an issue on the GitHub page.</p>
+<p>Known caveats:</p>
+<ul>
+  <li>Detection for features under alternative names is not yet supported</li>
+  <li>In older browsers, some attributes aren't initialized in the API prototypes, resulting in false negatives (see <a href="https://github.com/foolip/mdn-bcd-collector/issues/406">#406</a>)</li>
+  <li>Some APIs are accessed through other APIs and aren't exposed on their own; custom tests have not been written for them all</li>
+</ul>
+
+<p><a href="https://github.com/foolip/mdn-bcd-collector">Source on GitHub</a></p>
+
+<script src="/resources/json3.min.js"></script>
+<script>
+function getDropdownSelection(dropdown) {
+  var index = dropdown.selectedIndex;
+  var choice = dropdown.options[index].value;
+  return [index, choice];
+};
+
+function main() {
+  var button = document.getElementById('start');
+  var dropdown = document.getElementById('tests');
+  var individualTestOptionsBox = document.getElementById('individualTestOptionsBox');
+  var secureContext = document.getElementById('secureContext');
+  var limitExposure = document.getElementById('limitExposure');
+
+  dropdown.onchange = function() {
+    var choice = getDropdownSelection(dropdown);
+
+    if (choice[0] == 0) {
+      individualTestOptionsBox.style.display = "none";
+    } else {
+      individualTestOptionsBox.style.display = "block";
+    }
+
+    if (choice[1].indexOf('css') > -1) {
+      limitExposureBox.style.display = "none";
+    } else {
+      limitExposureBox.style.display = "block";
+    }
+  }
+
+  button.onclick = function() {
+    var choice = getDropdownSelection(dropdown);
+    var newUrl = choice[1];
+    if (choice[0] > 0) {
+      var protocol = secureContext.checked ? 'https:' : 'http:';
```

Comment:
Could this use the protocol of the current page instead? For localhost 'https:' will never make sense, so showing the option then isn't really useful.